### ISSUE TITLE
Support cloning into issue editor

### ISSUE THREAD

**Issue by knutwannheden** [gatewayapps/kamino#151]
I would (as an option) like to be able to clone an issue and be able to edit it in the GitHub issue editor before actually creating it.

**Comment by johnmurphy01** [comment]
@all-contributors please add @knutwannheden for ideas

**Comment by fharper** [comment]
Good idea! This is what I thought this extension would do, so I will look forward to this feature if implemented.
